spock-specs/src/test/groovy/org/spockframework/smoke/VerifyAllMethodsSpecification.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.spockframework.smoke
+
+import org.opentest4j.MultipleFailuresError
+import org.spockframework.EmbeddedSpecification
+import org.spockframework.runtime.ConditionNotSatisfiedError
+import org.spockframework.runtime.SpockComparisonFailure
+import spock.lang.VerifyAll
+
+class VerifyAllMethodsSpecification extends EmbeddedSpecification {
+
+  def setup() {
+    runner.throwFailure = false
+  }
+
+  @VerifyAll
+  private static void isPositiveAndEven(int x) {
+    x > 0
+    x % 2 == 0
+  }
+
+  def "@VerifyAll method succeeds"() {
+    expect:
+    isPositiveAndEven(2)
+  }
+
+  def "@VerifyAll method fails"() {
+    when:
+    def result = runner.runSpecBody '''
+@VerifyAll
+private static void isPositiveAndEven(int x) {
+    x > 0
+    x % 2 == 0
+}
+
+def "feature"() {
+    expect:
+    isPositiveAndEven(-3)
+}
+'''
+
+    then:
+    result.failures.size() == 1
+    with(result.failures[0].exception, MultipleFailuresError) {
```

Comment:
Isn't it
```suggestion
    with(result.failures[0].exception, SpockMultipleFailuresError) {
```

Reply:
It actually was not working properly yet, fixed in [`1858d85` (#2112)](https://github.com/spockframework/spock/pull/2112/commits/1858d857dc8cd455a1d6e103bfb76e11f2328efa)